src/pages/lib/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,11 +1,456 @@
+
+  return (
+    <Authenticator>
+      {({ signOut, user }) => (
+        <div className="App">
+          <div>
+            <pre>{JSON.stringify(user?.attributes, null, 2)}</pre>
+          </div>
+          {showValidationCodeUI === false && (
+            <button
+              onClick={async () => { 
+                await updateUserEmail();
+                setShowValidationCodeUI(true);
+              }}
+            >
+              Update Email
+            </button>
+          )}
+          {showValidationCodeUI === true && <ValidationCodeForm />}
+          <button onClick={signOut}>Sign out</button>
+        </div>
+      )}
+    </Authenticator>
+  );
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```javascript
+// App.js
+import { Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+import { Auth } from 'aws-amplify';
+import { useState } from 'react';
+
+async function updateUserEmail() {
+  try {
+    const user = await Auth.currentAuthenticatedUser();
+    await Auth.updateUserAttributes(user, {
+      email: 'updatedEmail@mydomain.com'
+    });
+
+    console.log('a verification code is sent');
+  } catch(err) {
+    console.log('failed with error', err);
+  }
+}
+
+async function verifyEmailValidationCode(code) {
+  try {
+    await Auth.verifyCurrentUserAttributeSubmit('email', code);
+    console.log('email verified');
+  } catch(err) {
+    console.log('failed with error', err);
+  }
+}
+
+function ValidationCodeForm() {
+  const [validationCode, setValidationCode] = useState(null);
+  return (
+    <div>
+      <label>
+        Verification Code (sent to the new email):
+        <input
+          onChange={(e) => {
+            setValidationCode(e.target.value);
+          }}
+          type="text"
+          name="vc"
+        />
+      </label>
+      <button onClick={() => verifyEmailValidationCode(validationCode)}>
+        Send Code
+      </button>
+    </div>
+  );
+}
+
+export default function App() {
+  const [showValidationCodeUI, setShowValidationCodeUI] = useState(false);
+
+  return (
+    <Authenticator>
+      {({ signOut, user }) => (
+        <div className="App">
+          <div>
+            <pre>{JSON.stringify(user.attributes, null, 2)}</pre>
+          </div>
+          {showValidationCodeUI === false && (
+            <button
+              onClick={async () => {
+                await updateUserEmail();
+                setShowValidationCodeUI(true);
+              }}
+            >
+              Update Email
+            </button>
+          )}
+          {showValidationCodeUI === true && <ValidationCodeForm />}
+          <button onClick={signOut}>Sign out</button>
+        </div>
+      )}
+    </Authenticator>
+  );
+}
+```
+</Block>
+</BlockSwitcher>
+
+</Accordion>
+
+### Delete user attributes
+
+Finally, if your users need to delete one or more user attributes, you can call `Auth.deleteUserAttributes` method of the `Auth` class to enable this action:
```

Comment:
Finally, if your users need to delete one or more user attributes, you can call `Auth.deleteUserAttributes` method: